dist/screeps.d.ts

```diff
@@ -1216,7 +1216,7 @@ interface PathFinder {
      *
      * @param isEnabled Whether to activate the new pathfinder or deactivate.
      */
-    use(isEnabled: boolean): any;
+    use(isEnabled: boolean): undefined;
```

Comment:
I recommend generally using `void` as the return type instead of `undefined` when the intent is for it to be an error to try and use the returned value from the function, despite that at runtime, `undefined` is indeed the value of the expression `PathFinder.use()`.